### DISCUSSION TITLE
Change default port for web page display

### DISCUSSION THREAD

**Discussion by dtaussigNV** [compiler-explorer/compiler-explorer#6584] [Q&A] [answered]
I want to run multiple instances of compiler-explorer on my local machine, each with a different port number for web browser interface.

Two questions:

1. Can I do that?
2. If yes, then how do I configure the port number for each instance?

**Comment by mattgodbolt** [comment, marked as answer]
1. Yes!
2. supply a different value for the `--port` command-line option

If you're running CE directly, adding a port command-=line option is easy. If you're running with `make`, then add it to the `EXTRA_ARGS` like: `make EXTRA_ARGS='--port 10242'`

**Reply by dtaussigNV** [reply]
Thanks Matt - did the trick
